### PR TITLE
Added support to both Old and new APIs

### LINKED ISSUE #1759: Support old and new API in current development branch
At some point the old API was taken out and now we have two branches that become incompatible more and more. Instead the dev branch should support both APIs, including the one in APIv1 branch.

### PR DESCRIPTION
**Pull request by agbilotia1998**
Fixes #1759
Link to review the changes: https://openeveweb.herokuapp.com/

### CODE REVIEW

**Comment by agbilotia1998** [comment]
@mariobehling @geekyd please review.

**Comment by mariobehling** [comment]
Works well for me.

@geekyd What do you think?

**Comment by agbilotia1998** [comment]
@geekyd done but the tests are failing due to `saucelabs` timeout issue on `brokenLinks` count test, I removed that test and the travis is passing.Please check https://travis-ci.org/fossasia/open-event-webapp/builds/351635906?utm_source=github_status&utm_medium=notification
So should I keep that test or remove that from the test suite?

**Comment by geekyd** [comment]
@agbilotia1998 The test is passing but there are a few codacy issues still pending. Please have a look at them

**Comment by agbilotia1998** [comment]
@geekyd the pending codacy issues are related to API keys we have used like 'license_url' those cannot be changed.Also, the tests are passing after removing the `check broken links in navbar and footer` test.

File: src/backend/generator.js [hunk 1]
```diff
@@ -12,9 +12,9 @@ const sass = require('node-sass');
 const jsonfile = require('jsonfile');
 const minify = require('html-minifier').minify;
 const distHelper = require(__dirname + '/dist.js');
-const fold = require(__dirname + '/fold.js');
 const mailer = require('./mailer');
 const ftpDeployer = require('./ftpdeploy');
+var fold;
```
Comment (anshumanv) [inline src/backend/generator.js]: Please use let

**Comment by mariobehling** [comment]
@geekyd I am merging this as work in progress. Let's work with everyone and get things merged faster to keep the flow.

**Pull request merged**